bin/fillTestData/Users.js

```diff
@@ -133,138 +134,172 @@ var addUsers = function () {
 
-                // Save the user
-                user.save(function (err) {
-                  savedUsers++;
-                  process.stdout.write('.');
-                  if (!err && admin!== undefined) {
-                    console.log('Created admin user. Login with: ' + admin + ' / password');
-                  } else if (err && admin !== undefined) {
-                    console.log(chalk.red('Could not add admin user ' + admin));
-                    console.log(err);
-                  } else if (err) {
-                    console.log(err);
-                  }
-
-                  addOffer(user._id, index, max, users.length, limit, done);
-                });
-
-
-                // No more admin users
-                if (numAdminUsers === 1) {
-                  printWarning();
+              // Save the user
+              user.save(function (err) {
+                savedUsers++;
+                process.stdout.write('.');
+
+                if (!err && admin!== undefined) {
+                  console.log('Created admin user. Login with: ' + admin + ' / password');
+                } else if (err && admin !== undefined) {
+                  console.log(chalk.red('Could not add admin user ' + admin));
+                  console.log(err);
+                } else if (err) {
+                  console.log(err);
                 }
 
-                if (admin !== undefined) {
-                  numAdminUsers--;
+                addOffer(user._id, index, max, users.length, limit, done);
+              });
+
+
+              // No more admin users
+              if (numAdminUsers === 1) {
+                printWarning();
+              }
+
+              if (admin !== undefined) {
+                numAdminUsers--;
+              }
+            }());
+
+            index++;
+          }
+        },
+
+        // Update tribes with the new tribe counts once all users have been  added
+        function updateTribes(done) {
+          var numTribesUpdated = 0;
+
+          // If we didn't add any users, tribes do not need to be updated
+          if (savedUsers === 0) {
+            done(null);
+          } else {
+            // Update tribes
+            for (var j = 0; j < tribes.length; j++) {
+              Tribe.findByIdAndUpdate(tribes[j]._id, tribes[j], function (err) {
+                if (err) {
+                  console.error(err);
                 }
-              }());
 
-              index++;
+                numTribesUpdated += 1;
+                if (tribes.length === numTribesUpdated) {
+                  done(null);
+                }
+              });
             }
-          }).catch(function (err) {
-            console.log(err);
-            done(err, null);
+          }
+        },
+
+        // disconnect from mongo
+        function disconnect(done) {
+          mongooseService.disconnect(function () {
+            done(null);
           });
         }
-      ]);
 
+      ]); // asyc.waterfall
```

Comment:
We can now start porting code to ES6 :-) :tada:

@mrkvon has been working on some docs here: https://github.com/Trustroots/trustroots/pull/1001

Reply:
Yes! Any readability feedback and what is missing is much appreciated. Still heavily a work in progress.

Reply:
@simison - I went ahead and made a good amount of ES6 changes to the User script, but was seeing a failure due to async/await syntax. Apparently, the `bin` directory had its own `.eslintrc.js` file which had not been updated to use `ecmaVersion2018`. 

I went ahead and made the change, but just want to be sure we don't need to do any overrides similar to what you did for the main eslint file. It didn't to appear to cause any issues in the other bin directory files, but just want to make sure I am not missing anything. 

Thanks! I can update the Tribes and Messages in another PR.

Reply:
>Apparently, the `bin` directory had its own `.eslintrc.js` file which had not been updated to use `ecmaVersion2018`.

Yep, that was by mistake — fixed in with https://github.com/Trustroots/trustroots/pull/1042 and ES6 is ok again in `bin/` :-)